### DISCUSSION TITLE
Is this transaction a success？

### DISCUSSION THREAD

**Discussion by Alteryy** [filecoin-project/lotus#6230] [General Q&A] [answered]
https://filfox.info/zh/message/bafy2bzaceb2u6t7sfcs3yxuzctgv556fmgermvjhzzxmdqrlnllrut7wpphiy

This transaction has no information and status, Is this transaction a success？

**Comment by moremorefun** [comment, marked as answer]
No.
It‘s replaced by https://filfox.info/zh/message/bafy2bzacecoj4ayajkc7mudglsheq7n3yalyfeq7jytny63ylmfbmuxxwgiq4
They have the same nonce.
